.packit.yaml

```diff
@@ -87,6 +87,21 @@ jobs:
     owner: psss
     project: tmt
 
+  # Release to copr
+  - job: copr_build
+    trigger: release
+    branch: main
+    targets:
+      - fedora-all
+      - epel-9
+    enable_net: False
+    list_on_homepage: True
+    preserve_project: True
+    owner: psss
+    project: tmt
+    actions:
+        post-upstream-clone: []
```

Comment:
The thing is that we replace each config key completely => the `actions` dictionary is not merged with the top-level one, but is completely replaced.

(Some teams rely on this so we can't change this behaviour easily..)

Reply:
Ah, I see, thanks for clarification. Should be fixed by e1106e4f.